src/camvidlog2/queries.py

```
@@ -3,42 +3,46 @@
-            raise TypeError("Unexpected item in embedding_group")
+    for embedding_group in embedding_collection.groups:
+        for item in embedding_group.items:
+            if isinstance(item, StringEmbedding):
+                nda_combined_list.append(nda_strings[string_count, :])
+                string_count += 1
+            elif isinstance(item, FrameEmbedding):
+                nda_combined_list.append(
+                    frame_embedding_data.loc[(str(item.filepath), item.frame_no), :],
+                )
+            else:
+                raise TypeError("Unexpected item in embedding_group")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)